src/test/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImplTest.java

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
Not to be solved in this PR, but shouldn't this class be named CkanBackendImplTest instead of CKANBackendImplTest? :)